tests/e2e/cucumber/environment/index.ts

```diff
@@ -106,6 +106,10 @@ After(async function (this: World, { result }: ITestCaseHookParameter) {
   if (result.status !== Status.PASSED) {
     await this.actorsEnvironment.close()
     spaceStore.clear()
+    linkStore.clear()
+  } else {
+    linkStore.clear()
+    spaceStore.clear()
   }
```

Comment:
```suggestion
  if (result.status !== Status.PASSED) {
    await this.actorsEnvironment.close()
    }
    spaceStore.clear()
    linkStore.clear()
```
IMO we can do this if `spaceStore.clear()` and `linkStore.clear()` are executing either ` if (result.status !== Status.PASSED) ` is `true` or `false`. What you say?